README.md

```diff
@@ -6,7 +6,7 @@ nakedret is a Go static analysis tool to find naked returns in functions greater
 Install Nakedret via go install:
 
 ```cmd
-go install github.com/alexkohler/nakedret@latest
+go install github.com/alexkohler/nakedret/cmd/nakedret@latest
```

Comment:
From some local testing, it seems like the command-line tool now has to be installed with
`go install github.com/alexkohler/nakedret/cmd/nakedret` (with cmd/nakedret added)
but this is the same as for example goimports
`go install golang.org/x/tools/cmd/goimports@latest`

I'm not really sure what are best practices for things like this, it looks like many projects use both cmd/x and the root directory. goimports and golangci-lint both use cmd/x as well

Reply:
> From some local testing, it seems like the command-line tool now has to be installed with
go install github.com/alexkohler/nakedret/cmd/nakedret (with cmd/nakedret added)

Hm, would this break folks that are running things like `go install github.com/alexkohler/nakedret@latest` in CI? (not saying that's the best practice ever :sweat_smile:)

> I'm not really sure what are best practices for things like this, it looks like many projects use both cmd/x and the root directory. goimports and golangci-lint both use cmd/x as well

Seems like `cmd/x` might be the most idiomatic thing to do, but I don't really have strong feelings - I'd be fine with `cmd/` or `cmd/x` :+1:

Reply:
> Hm, would this break folks that are running things like go install github.com/alexkohler/nakedret@latest in CI? (not saying that's the best practice ever 😅)

that's true, but I guess then you would have the option to either use a specific version if you don't want to update anything, or update the reference?

maybe even update the README to refer to golangci-lint as the primary way to run it? it's just so much easier to all kinds of linting in one place… still good to be able to run each individual linter as a tool of course!

Reply:
Yeah, I think most workflows should be using `golangci-lint`. I think this is a positive change that'll reduce some tech debt over in `golangci-lint`, so even if this ends up breaking some `go install github.com/alexkohler/nakedret@latest` calls, it's probably worth it IMO.